rpc_jobs/multi_node_aio.yml

```diff
@@ -190,3 +206,145 @@
             common.delete_workspace()
         }}
       }} // cit node
+
+- job-template:
+    name: 'Leapfrog-OnMetal-Multi-Node-AIO_{series}-{context}-{trigger}'
+    project-type: workflow
+    properties:
+      - build-discarder:
+          days-to-keep: 7
+      - rpc-openstack-github
+    concurrent: true
+    triggers:
+      - timed: "{CRON}"
+    parameters:
+      - kibana_selenium_params:
+          KIBANA_SELENIUM_BRANCH: "{branch}"
+      - rpc_gating_params
+      - instance_params:
+          IMAGE: "OnMetal - Ubuntu 14.04 LTS (Trusty Tahr)"
+          FLAVOR: "onmetal-io1"
+          REGION: "IAD"
+      - rpc_repo_params:
+          RPC_BRANCH: "{branch}"
+      - osa_ops_params:
+          DEFAULT_IMAGE: "{DEFAULT_IMAGE}"
+          DEFAULT_KERNEL: "3.13.0-98"
+          DATA_DISK_DEVICE: "sdb"
+      - text:
+          name: "USER_VARS"
+          default: "{USER_VARS}"
+          description: "OSA/RPC USER_VARS to inject for this build"
+      - string:
+          name: STAGES
+          default: |
+            Allocate Resources, Connect Slave, Prepare Multi-Node AIO,
+            Prepare RPC Configs, Deploy RPC w/ Script, Prepare MaaS,
+            Setup MaaS, Verify MaaS, Install Tempest, Tempest Tests, Holland,
+            Leapfrog Upgrade, Install Tempest, Tempest Tests, Holland,
+            Verify MaaS, Prepare Kibana Selenium, Kibana Tests, Cleanup,
+            Destroy Slave
+          description: |
+            Pipeline stages to run CSV. Note that this list does not influence execution order.
+            Options:
+              Allocate Resources
+              Connect Slave
+              Prepare Multi-Node AIO
+              Prepare RPC Configs
+              Deploy RPC w/ Script
+              Prepare MaaS
+              Setup MaaS
+              Verify MaaS
```

Comment:
Not sure we need to verify maas, tempest tests and holland  before leapfrog. It should be already tested elsewhere. (Kilo deploy on multi node). If kilo deploy on multi node doesn't exist yet, we should build it.